docker-image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7
+
+RUN apt-get update 
+
+RUN apt-get install -y cmake
+
+RUN apt-get install -y git
+
+RUN git clone https://github.com/neomatrix369/nlp_profiler.git
```

Comment:
Instead of doing this try to see if you can actually use the local folder and the code downloaded in it - there is an advantage and purpose for doing it.

See how it has been done [here](https://github.com/neomatrix369/awesome-ai-ml-dl/tree/master/examples/data/databases/graph/grakn).

Reply:
Thank you for the suggestion Mani. I've updated the file so it uses the local folder.